src/views.rs

```diff
@@ -103,6 +103,7 @@ pub struct EncodableCrate {
     pub repository: Option<String>,
     pub links: EncodableCrateLinks,
     pub exact_match: bool,
+    pub audit_actions: Vec<EncodableAuditAction>,
```

Comment:
This publicly exposes the audit information via the API, right? We need to discuss whether this should be public. In any case, if this is available via the API, it should also be included in the database dumps and vice versa. Currently this PR excludes the information from the DB dumps.

Reply:
Once we resolve the issue I'll update this, and for version owner actions. I wasn't that familiar with what's expected here.